pkg/time/rate/rate_test.go

```diff
@@ -182,6 +182,9 @@ func TestLongRunningQPS(t *testing.T) {
 		t.Skip("low resolution time.Sleep invalidates test (golang.org/issue/14183)")
 		return
 	}
+	if runtime.GOOS == "windows" {
+		t.Skip("test is unreliable on windows")
+	}
```

Comment:
This file is stolen from golang sources with some minor modification. Is this not windows compatible ? It may even be worth checking the newer version of golang if it needs updating.